en/drafts/originals/ocr-with-google-vision-and-tesseract.md

```diff
@@ -24,92 +24,92 @@ doi: tbc
 
-This method usually performs well, but it still relies on Google Vision for layout detection. Although the vertical stacking of the text regions significantly helps reduce errors, it is still possible for mistakes to appear, especially if you have many small text regions in your documents. A drawback of this method is that any mapping from the the source facsimile/PDF to the resulting text is lost. 
+This method usually performs well, but it still relies on Google Vision for layout detection. Although the vertical stacking of the text regions significantly reduces errors, it is still possible for mistakes to appear, especially if you have many small text regions in your documents. A drawback of this method is that any mapping from the source facsimile/PDF to the resulting text is lost.
 
 #### Second combined method
 
 The second combined method works with the original PDF but, instead of using the OCR text string that Google Vision provides for each page, it searches the JSON output files for the words that fall within the bounds of the text regions identified by Tesseract. This is possible because Google Vision provides coordinates for each word in the document.
 
 This method has the advantage of not relying on Google Vision's layout detection at all. However, the downside is that line breaks that were not initially identified by Google Vision cannot be easily reintroduced. Therefore, if it is important for your project that the OCRed text retains line breaks at the correct locations, the first combined method will be the best choice.
 
-The following three examples highlight the potential benefits of using Google Vision, Tesseract, or one of the combined methods. Each image represents two pages from the dataset we will be using in this lesson. The outputs created for the passages highlighted in yellow by each of the four methods are detailed in the table below each image.
+The following three examples highlight the potential benefits of using Google Vision, Tesseract, or one of the combined methods. Each image represents two pages from the dataset we will be using in this lesson. Outputs created for the passages highlighted in yellow by each of the four methods are detailed in the table below each image.
 
 ## Comparisons
 
 ### Example 1
 
-{% include figure.html filename="ocr-with-google-vision-and-tesseract1.png" caption="Figure 1: First two pages of “Tomb of King Henry IV in Canterbury Cathedral”, with four highlighted lines indicating the text used in the OCR results below." %}
+{% include figure.html filename="ocr-with-google-vision-and-tesseract1.png" alt="Visual description of figure image" caption="Figure 1: First two pages of \“Tomb of King Henry IV in Canterbury Cathedral\”, with four highlighted lines indicating the text used in the OCR results below." %}

```

Comment:
Hello @isag91. As part of our commitment to accessibility, we are now asking authors to provide descriptive alt-text to accompany figure images. 

You can slot the alt-text in here.

I've collated [some resources on our Wiki](https://github.com/programminghistorian/jekyll/wiki/Achieving-Accessibility-Alt-text-Colour-Contrast#alt-text) as guidance if you need it.